packages/ui/package.json

```diff
@@ -125,6 +125,7 @@
     "rehype-rewrite": "^4.0.2",
     "rehype-sanitize": "^6.0.0",
     "rehype-video": "^2.0.2",
+    "remark-breaks": "^4.0.0",
```

Comment:
@praneshg239 is this a new library? If yes, we need to get it vetted first.

Reply:
Added a ticket for approval:
https://harness.atlassian.net/browse/SECENG-817